presto-main/src/test/java/com/facebook/presto/server/TestServer.java

```diff
@@ -85,6 +89,38 @@ public void teardown()
         Closeables.closeQuietly(client);
     }
 
+    @Test(timeOut = 10000)
+    public void testServerStartsWithGC()
+            throws Exception
+    {
+        // force a GC
+        long gcCount = getNumGcs();
+        System.gc();
+        while (gcCount >= getNumGcs()) {
+            Thread.sleep(1000);
+            System.gc();
```

Comment:
This may or may not cause the full gc. There are JVM flags that can turn it to only trigger a concurrent collection, or even completely ignored. I would simply remove this tests, as effectively it just verifies the MX bean correctness.

Reply:
I don't know how this change is here. It was removed from the original PR.